tests/unit/acquisition/test_function.py

```diff
@@ -85,14 +92,41 @@ def test_expected_improvement_builder_builds_expected_improvement(
     npt.assert_array_almost_equal(acq_fn(query_at), expected)
 
 
-def test_expected_improvement() -> None:
-    def _ei(x: tf.Tensor) -> tf.Tensor:
-        n = tfp.distributions.Normal(0, 1)
-        return - x * n.cdf(-x) + n.prob(-x)
+@random_seed()
+@pytest.mark.parametrize('best, rtol, atol', [
+    (tf.constant([50.0]), 0.01, 1e-3),
+    (BRANIN_GLOBAL_MINIMUM[None], 0.01, 1e-3),
+    (BRANIN_GLOBAL_MINIMUM[None] * 1.01, 0.01, 1e-3)
+])
+# todo this fails for variance scale 100. is that ok?
+@pytest.mark.parametrize('variance_scale', tf.constant([0.1, 1.0, 10.0], dtype=tf.float64))
```

Comment:
i need to make the rtol 0.03 for the last two tests if I add variance scale 100. Is that a problem?

Reply:
I don't understand what variance scale 100 means

Reply:
it's the `amplitude` argument to the Matern52 kernel in the model, so determines the size of the model variance

Reply:
I'd choose: amplitude=100, rtol=0, atol=1e-2. atol can be increased to 1e-1 if the test doesn't pass

Reply:
it fails with 1e-2 and 1e-1